app/util/requests.ts

```diff
@@ -82,3 +81,20 @@ export function getChecksum(filePath: string) {
     file.on('error', e => reject(e));
   });
 }
+
+export function jfetch<TResponse>(request: RequestInfo, init?: RequestInit): Promise<TResponse> {
```

Comment:
should we cast the default type to `unknown`?
`jfetch<TResponse=unknown>(..`

Reply:
Yeah I originally had that, but the default type seemed to already be `unknown`.  I'm not sure how TypeScript decides the default type.  It can't hurt to be explicit though.